testbed/lib/main.dart

```diff
@@ -31,13 +31,13 @@ void main() {
 
   // Try to resize and reposition the window to be half the width and height
   // of its screen, centered horizontally and shifted up from center.
-  if (Platform.isMacOS) {
+  if (Platform.isMacOS || Platform.isLinux) {
     window_size.getWindowInfo().then((window) {
       if (window.screen != null) {
         final screenFrame = window.screen.visibleFrame;
-        final width = math.max((screenFrame.width / 2).roundToDouble(), 640.0);
+        final width = math.max((screenFrame.width / 2).roundToDouble(), 800.0);
```

Comment:
This change sees unrelated to the rest of the PR, is that true?

Reply:
Not exactly. When I wrote this code on macOS to demo/exercise the new plugin, I forgot that `testbed` was 800x600, not 640x480. While testing it on Linux on a small screen with low resolution, I realized that this would actually resize testbed to be smaller on some configurations, which wasn't intended.

Reply:
Makes sense. Thanks! Resolving.